src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f' ? 'husband' : 'wife';
```

Comment:
The logic for determining the partner's role is incorrect. It should be `const partner = person.sex === 'f' ? 'wife' : 'husband';` to correctly assign 'wife' to females and 'husband' to males.

Reply:
But females have husbands and vice versa